shared/desktop/package.js

```diff
@@ -1,6 +1,7 @@
 // @flow
 import del from 'del'
 import fs from 'fs-extra'
+import klawSync from 'klaw-sync'
```

Comment:
so fs-extra copy doesnt' filter per file anymore, so if you pass a path it just passes the full pathname to the filter. not what we want. instead we 'walk' the hierarchy and do the filtering ourselves